cmd/ova-provider-server/ova-provider-server.go

```diff
@@ -429,6 +431,45 @@ func readOVF(ovfFile string) (*Envelope, error) {
 	return &envelope, nil
 }
 
+const (
+	Unknown    = "Unknown"
+	VMware     = "VMware"
+	VirtualBox = "VirtualBox"
+	RHV        = "RHV"
```

Comment:
Please switch to upstream oVirt to keep it consistent with other parts of Forklift.

Reply:
Done.